Add basic explanation to ProgrammableAndroids section. (#106)
Slimefun/Wiki: pages/Farmer-Androids.md

> A specialized android is created to automate farming process.

Reviewer: ```suggestion
An Android that specializes in automating the farming process.
```

> This android only works with planted seed and will not work on melon or pumpkin.

Reviewer: ```suggestion
This android only works with planted seeds and will not work with melons or pumpkins.
```

> This android only works with planted seeds and will not work with melons or pumpkins.

Reviewer 2: or sugar cane, or cactus.
We should probably add a table here with the seeds that **do** work.

## Obtaining
Farmer Androids can be crafted in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).

## Usage

> Farmer Androids can be programmed to move and to harvest and replants.

Reviewer: ```suggestion
Farmer Androids can be programmed to harvest or/and plant crops.
```
moving is practically harvesting and then just replanting at a diff location

> To power Androids you need compatible fuel source on each tiers.

Reviewer: ```suggestion
Androids need fuel in order to work, you have to fill them with the [Required Fuel](https://github.com/Slimefun/Slimefun4/wiki/Normal-Androids#power-source) before booting!
```

## Tiers

> There are two tiers on farmer android, each with different modifiers listed below.

Reviewer: ```suggestion
There are two tiers on the Farmer Android, each with different modifiers listed below.
```

> | Tier | Fuel Efficiency | Special Function |
> | ---- | --------------- | ---------------- |
> | Basic | 1.0x | None |
> | Advanced | 1.5x | Can harvest from Exotic Garden addons |

Reviewer: ```suggestion
| Tier     | Fuel Efficiency | Special Function                      |
| -------- | :-------------: | ------------------------------------- |
| Basic    | 1.0x            | None                                  |
| Advanced | 1.5x            | Can harvest from Exotic Garden addons |
```
